Jenkinsfile

```diff
@@ -366,6 +366,7 @@ pipeline {
             branch 'master'
             beforeAgent true
           }
+          options { timeout(time: 120, unit: 'MINUTES') }
```

Comment:
```suggestion
          options { timeout(time: 150, unit: 'MINUTES') }
```

Reply:
@dwightguth you can apply this change if you want a longer timeout. 2hrs should be enough in general, but padding it with 30min should help.